releases/release-1.31/release-notes/maps/pr-124075-map.yaml

```
@@ -0,0 +1,4 @@
+pr: 124075
+releasenote:
+  text: 'DRA: client-side validation of a ResourceHandle would have accepted a missing
```

```suggestion
  text: 'Dynamic Resource Allocation (DRA): client-side validation of a ResourceHandle would have accepted a missing
```

https://github.com/kubernetes/sig-release/pull/2542#discussion_r1653192050